### PR TITLE
Fix ClassCastException: RemoteWebElement cannot be cast to Locatable.

### PR DESCRIPTION
**Pull request by johnjohndoe**

### CODE REVIEW

File: src/TestFabric.java [hunk 1]
```diff
@@ -37,8 +36,7 @@ public static void main(String[] args) throws InterruptedException {
 		
 		// Scroll down to where you can see the "View all sessions" button
 		WebElement viewAllSessions = driver.findElement(By.linkText(CRASHLYTICS_VIEW_ALL_SESSIONS_LINK));
-		Locatable element = (Locatable) viewAllSessions;
-		Point p = element.getCoordinates().onPage();
+		Point p = viewAllSessions.getLocation();
```
Comment (krshmbb) [inline src/TestFabric.java:39]: Is there a particular version of Selenium web driver / Chrome where you are seeing this exception ?

Comment (johnjohndoe) [reply src/TestFabric.java:39]: I used the latest version available.

Comment (johnjohndoe) [reply src/TestFabric.java:39]: I used Selenium web driver version 3.8.1 from [here](http://www.seleniumhq.org/download/) and ChromeDriver version 2.34 from [here](https://sites.google.com/a/chromium.org/chromedriver/downloads).

Comment (johnjohndoe) [reply src/TestFabric.java:39]: @krshmbb Do you still have open questions?

Comment (krshmbb) [reply src/TestFabric.java:39]: Nope. Merged. Thanks for pushing up the fix!

**Pull request merged**